lib/config/config.js

```diff
@@ -439,6 +454,76 @@ function loadConfigFile (configFile) {
   return null
 }
 
+function printLoggingPreamble (config, sources, configFilePath) {
+  const { logger } = config
+  const nonDefaultKeys = ['environment', 'start', 'file']
+  const nonDefaultOpts = Object.assign.apply(null, nonDefaultKeys.map(k => sources[k] || {}))
+  const isPreview = AGENT_VERSION.indexOf('-') !== -1
+
+  // Version and pre-release warning
+  logger.info(`Elastic APM Node.js Agent, version: ${AGENT_VERSION}`)
+
+  if (isPreview) {
+    logger.warn('This is a pre-release version and not intended for use in production environments!')
+  }
+
+  // Environment
+  const offsetHours = -((new Date()).getTimezoneOffset() / 60)
+  const offsetSign = offsetHours >= 0 ? '+' : '-'
+  const offsetPad = Math.abs(offsetHours) < 10 ? '0' : ''
+  const timeZone = `UTC${offsetSign}${offsetPad}${Math.abs(offsetHours * 100)}`
+  const frameWorkName = nonDefaultOpts.frameWorkName || 'unknown'
+  const frameWorkVersion = nonDefaultOpts.frameWorkVersion || ''
+  const environmentMessage = [
+    '\nEnvironment:',
+    `- Process ID:  ${process.pid}`,
+    `- Process name: ${process.title}`,
+    // TODO: check sec concerns
+    // ` - command line: ${process.cwd()}`,
```

Comment:
This is also mentioned in the spec. Pointing to [this issue](https://github.com/elastic/apm-agent-nodejs/issues/1916)
Maybe it's better to not log this value